spec/tictactoe/input_output_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "./lib/tictactoe/output_input"
+require "./spec/tictactoe/fakers"
+
+RSpec.describe Tictactoe::OutputInput do
+  context "output" do
+    it "should output a message passed to it" do
+      io=Tictactoe::OutputInput.new
+      expect{io.output("yeah")}.to output("yeah\n").to_stdout
```

Comment:
This is another opportunity to inject something that wraps around `puts` , so that you could verify the output without checking `to_stdout`.